dttools/src/md5.c

```diff
@@ -367,6 +369,88 @@ const char *md5_string(unsigned char digest[16])
 	str[32] = 0;
 	return str;
 }
+const char * md5_dir_rec(char * src){
```

Comment:
It seems there is a combination of spaces and tabs when indenting these functions. Just make sure the style matches the rest of the file (tabs).

Reply:
Fixed.